docs/docs/single-tenant/fastapi_configuration.mdx

```diff
@@ -185,23 +185,46 @@ We've used two flags: `usePkceWithAuthorizationCodeGrant`, which is the authenti
 
 Now, the fun part begins! 🚀
 
-Import the `SingleTenantAzureAuthorizationCodeBearer` from `fastapi_azure_auth` and configure it:
+Import the `SingleTenantAzureAuthorizationCodeBearer` from `fastapi_azure_auth` and configure it. You can use the `Settings` class to compute the required URLs:
```

Comment:
nitpick but we also compute other properties, so maybe "... compute the required fields"